### PR TITLE
[TIMOB-25029] Removed generation of legacy launch images for iOS 6.

### PR DESCRIPTION
**Pull request by cb1kenobi**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-25029

### CODE REVIEW

File: iphone/cli/commands/_build.js [hunk 1]
```diff
@@ -5400,22 +5400,16 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
...
 					'Default-568h@2x.png':           { idiom: 'iphone', extent: 'full-screen', minSysVer: '7.0', orientation: 'portrait', width: 640, height: 1136, scale: 2, subtype: 'retina4' },
-					// iPhone Portrait - iOS 8,9 - Retina HD 4.7 (750x1334) iPhone 6
-					'Default-667h@2x.png':           { idiom: 'iphone', extent: 'full-screen', minSysVer: '8.0', orientation: 'portrait', width: 750, height: 1334, scale: 2, subtype: '667h' },
 
 					// iPad Landscape - iOS 7-9 - 1x (1024x768)
 					'Default-Landscape.png':         { idiom: 'ipad',   extent: 'full-screen', minSysVer: '7.0', orientation: 'landscape', width: 1024, height: 768, scale: 1 },
 					// iPad Landscape - iOS 7-9 - 2x (2048x1536)
 					'Default-Landscape@2x.png':      { idiom: 'ipad',   extent: 'full-screen', minSysVer: '7.0', orientation: 'landscape', width: 2048, height: 1536, scale: 2 },
-					// iPhone Landscape - iOS 8,9 - Retina HD 5.5 (2208x1242)
-					'Default-Landscape-736h@3x.png': { idiom: 'iphone', extent: 'full-screen', minSysVer: '8.0', orientation: 'landscape', width: 2208, height: 1242, scale: 3, subtype: '736h' },
```
Comment (hansemannn) [inline iphone/cli/commands/_build.js]: These ones are for iOS 8/9 and are used on the iPhone 6 / iPhone 6S / iPhone 7. Did you test the changes? I highly doubt these are the correct ones to remove.

**Review by hansemannn — CHANGES_REQUESTED** [review]

File: iphone/cli/commands/_build.js (hunk 1 above)
Comment (cb1kenobi) [inline iphone/cli/commands/_build.js]: Yes, I tested it. I know. Doesn't make sense, but those 3 are the culprit.

Comment (hansemannn) [inline iphone/cli/commands/_build.js]: Has to be an Xcode issue then. Those are the most recent launch-screens introduced by Apple, all newer devices would not have a proper launch-screen and scale-up the other ones. Need to dive more into this.

**Comment by hansemannn** [comment]
PR **rejected**, it removes required launch-images for the iPhone 6 / 7 and iPhone 6 Plus / 7 Plus. The initial warning from the linked ticket has to be an Xcode false-positive and will be tracked with Apple.

**Pull request closed without merging**